roles/cadvisor/meta/argument_specs.yml

```diff
@@ -0,0 +1,49 @@
+---
+# yamllint disable rule:line-length
+argument_specs:
+  main:
+    short_description: "cAdvisor"
+    description:
+      - "Deploy L(cadvisor,https://github.com/google/cadvisor) using ansible"
+    author:
+      - "cAdvisor Authors"
```

Comment:
I think this field is for the role authors

Reply:
Oh, ok.